### ISSUE TITLE
Error: 

### ISSUE THREAD

**Issue by bdarbonneau** [microsoft/Analysis-Services#73]
Using:  ALM toolkit v 5.0.7 , Power BI Desktop v 2.84.981.0 (latest on Sept 4)

After comparing datasets in Power BI Desktop and Power BI Service, I click on "Validate Selection" and get the following error:
 "System error: A new guard page for the stack cannot be created"

![image](https://user-images.githubusercontent.com/32508734/92213033-2f562780-ee93-11ea-9417-4f0fed8f7560.png)

**Comment by christianwade** [comment]
Hi Bertrand, this one has not been raised before. I have a few questions:
- Do you always get this error or is it intermittent?
- Do you get this error just for certain models, or for any model? If just for certain models and you're able to share with me by e-mail, please let me know. I'd like to get a repro.

Thanks,
Christian

**Comment by bdarbonneau** [comment]
Hi Christian,

It does not happen with every data model. Apparently with only one so far.
The difference only consisted in hiding a couple of columns.

[cid:image002.jpg@01D682EC.C935FE60]

[cid:image004.jpg@01D682EC.C935FE60]

I have copied each panel to a the attached text file, as I could not get the “Report Differences” to Excel to work (separate issue not reported!)

Unfortunately, the data is sensitive so I have only included the pbit.
Hoping that this will help.

rgds
Bertrand

**Comment by bdarbonneau** [comment]
And with that model, it happens every time, even I close/reopen pbd. Not intermittent.

From: D'ARBONNEAU Bertrand
Sent: 04 September 2020 18:56
To: 'microsoft/Analysis-Services' <reply@reply.github.com>
Subject: RE: [microsoft/Analysis-Services] Error: (#73)

Hi Christian,

It does not happen with every data model. Apparently with only one so far.
The difference only consisted in hiding a couple of columns.

[cid:image002.jpg@01D682EC.C935FE60]

[cid:image004.jpg@01D682EC.C935FE60]

I have copied each panel to a the attached text file, as I could not get the “Report Differences” to Excel to work (separate issue not reported!)

Unfortunately, the data is sensitive so I have only included the pbit.
Hoping that this will help.

rgds
Bertrand
